pom.xml

```diff
@@ -1777,7 +1784,7 @@
     <cxfVersion>3.5.5</cxfVersion>
     <cxfXjcVersion>3.3.2</cxfXjcVersion>
     <dhcp4javaVersion>1.1.0</dhcp4javaVersion>
-    <dnsjavaVersion>3.5.3_1</dnsjavaVersion>
+    <dnsjavaVersion>2.1.9_1</dnsjavaVersion>
```

Comment:
Why do we need to downgrade here? The last release of 2.x was back in 2019 whereas 3.5.3 is the last release available from end of 2023.

Reply:
According to mvnrepository.com this seems to be the last version without any CVEs.

Reply:
I was having problems with dependencies in Karaf not coming up correctly with the newer DNSJava, I have no idea why. I spent a day trying to figure it out and just said "f**k it, I'll put it back" and then everything started OK. _shrug_